Dockerfile default (#23)
MetroStar/ib_manifest_util: README.md

## Overview

Updating packages for a docker image in the Iron Bank Image Repository can be
time consuming since it requires that you rebuild multiple files and run
multiple separate processes. This tool captures all those tasks and provides

> a single function that needs to be run to update all the necessary files for
> updating.

Reviewer: ```suggestion
a single function that can be run to update all the necessary files.
```

)
```

> From there, all the files you need to update the repo will be generated

Reviewer: ```suggestion
From there, all the files you need to update in the repo will be generated
```
